src/Bridge/Symfony/Validator/EventListener/ValidationExceptionListener.php

```diff
@@ -44,12 +46,22 @@ public function onKernelException(ExceptionEvent $event): void
         if (!$exception instanceof ValidationException) {
             return;
         }
+        $exceptionClass = \get_class($exception);
+        $statusCode = Response::HTTP_BAD_REQUEST;
```

Comment:
Maybe should we default to `422` here to be as correct as possible by default?

Reply:
@dunglas  I appreciate your feedback, I definitely agree with you. Just are you ok with updating the http code for GraphQL ? https://github.com/api-platform/core/blob/master/src/GraphQl/Serializer/Exception/ValidationExceptionNormalizer.php#L42

Reply:
@dunglas  I appreciate your feedback, I definitely agree with you. Just, are you ok with also updating the http code for GraphQL ? https://github.com/api-platform/core/blob/master/src/GraphQl/Serializer/Exception/ValidationExceptionNormalizer.php#L42

Reply:
It looks good to me, wdyt @alanpoulain?